app/Lib/Rscripts/analyse.R

```diff
@@ -2,11 +2,31 @@
 library(psy)
 
 Analyse <- function(key, input.answers, number.answeroptions) {
+  # Calculates standard psychometric properties of an exam, notably the percentage correct and the item rest correlation (IRC) per item and 
+  # per answer option and the cronbach's alpha for the whole exam.
+  #
+  # Arguments:
+  #   key: Matrix of 0's and 1's. key[i,j] implies wether answer option i to item j is right (1) or wrong (0). If a row (item) consists of
+  #        only 0s, the item is interpreted as graded manually.
+  #   input.answers: Ungraded matrix of answers. input.answers[i,j] is the answer of student (i) to item (j).
+  #   number.answersoptions: Vector with number of answer options per item, should be equal to number of collumns of input.answers.
+  # 
+  # Returns:
+  #  list with:
+  #   Cronbach's alpha 
+  #   Maximum number of answer options
+  #   Vector of number of correct per item
+  #   Vector of percentage correct per item
+  #   Vector of IRC per item
+  #   Matrix[i,j] of number of students answering option i  to item j
+  #   Matrix[i,j] of percentage of students answering option i  to item j
+  #   Matrix[i,j] of IRC of answer option i to item j
+
   number.students <- nrow(input.answers)
   number.questions <- ncol(input.answers)
 
-  if (number.questions > 1 & number.students > 1) {
-    # Correct/Incorrect Matrix
+  if (number.questions > 2 & number.students > 1) { # Do the analysis only if there are at least 3 items and 2 students.
```

Comment:
Check the Google's R Style Guide: https://google.github.io/styleguide/Rguide.xml#comments